Sources/TuistKit/Services/InitService.swift

```diff
@@ -136,6 +161,15 @@ class InitService {
         }
     }
 
+    /// Checks if string is URL valid format.
+    ///
+    /// - Parameter string: string to be checked
+    /// - Returns: Bool
+    private func isURLFormattedString(_ string: String) -> Bool {
```

Comment:
Nitpick: I'd extract this one into [this file](https://github.com/tuist/tuist/blob/main/Sources/TuistSupport/Extensions/String+Extras.swift) and add a unit test. You can turn it into a property:

```swift
extension String {
 var isURL: Bool = {
  ...
 }
}
```